Server/Includes/Data.hpp

```diff
@@ -0,0 +1,54 @@
+#ifndef DATA_HEADER
+#define DATA_HEADER
+
+namespace data {
+    constexpr const char* const raw_mem_cert =
+        "-----BEGIN CERTIFICATE-----\n"
+        "MIICpjCCAZCgAwIBAgIESEPtjjALBgkqhkiG9w0BAQUwADAeFw0wODA2MDIxMjU0\n"
+        "MzhaFw0wOTA2MDIxMjU0NDZaMAAwggEfMAsGCSqGSIb3DQEBAQOCAQ4AMIIBCQKC\n"
+        "AQC03TyUvK5HmUAirRp067taIEO4bibh5nqolUoUdo/LeblMQV+qnrv/RNAMTx5X\n"
+        "fNLZ45/kbM9geF8qY0vsPyQvP4jumzK0LOJYuIwmHaUm9vbXnYieILiwCuTgjaud\n"
+        "3VkZDoQ9fteIo+6we9UTpVqZpxpbLulBMh/VsvX0cPJ1VFC7rT59o9hAUlFf9jX/\n"
+        "GmKdYI79MtgVx0OPBjmmSD6kicBBfmfgkO7bIGwlRtsIyMznxbHu6VuoX/eVxrTv\n"
+        "rmCwgEXLWRZ6ru8MQl5YfqeGXXRVwMeXU961KefbuvmEPccgCxm8FZ1C1cnDHFXh\n"
+        "siSgAzMBjC/b6KVhNQ4KnUdZAgMBAAGjLzAtMAwGA1UdEwEB/wQCMAAwHQYDVR0O\n"
+        "BBYEFJcUvpjvE5fF/yzUshkWDpdYiQh/MAsGCSqGSIb3DQEBBQOCAQEARP7eKSB2\n"
+        "RNd6XjEjK0SrxtoTnxS3nw9sfcS7/qD1+XHdObtDFqGNSjGYFB3Gpx8fpQhCXdoN\n"
+        "8QUs3/5ZVa5yjZMQewWBgz8kNbnbH40F2y81MHITxxCe1Y+qqHWwVaYLsiOTqj2/\n"
+        "0S3QjEJ9tvklmg7JX09HC4m5QRYfWBeQLD1u8ZjA1Sf1xJriomFVyRLI2VPO2bNe\n"
+        "JDMXWuP+8kMC7gEvUnJ7A92Y2yrhu3QI3bjPk8uSpHea19Q77tul1UVBJ5g+zpH3\n"
+        "OsF5p0MyaVf09GTzcLds5nE/osTdXGUyHJapWReVmPm3Zn6gqYlnzD99z+DPIgIV\n"
+        "RhZvQx74NQnS6g==\n"
+        "-----END CERTIFICATE-----\n";
+
+    constexpr const char* const raw_mem_key =
```

Comment:
The key should be private, otherwise I could make my own certificate with it.

Reply:
I should probably add a note to look at Main.cpp somewhere in there or explain that its only used if a private key / public crt wasn't found.

Reply:
```cpp
    // Set server crt
    if (std::filesystem::is_regular_file(so.MemCert())) {
        cw.https_mem_cert(so.MemCert());
    } else {
        cw.raw_https_mem_cert(data::raw_mem_cert);
    }

    // Set server key
    if (std::filesystem::is_regular_file(so.MemKey())) {
        cw.https_mem_key(so.MemKey());
    } else {
        cw.raw_https_mem_key(data::raw_mem_key);
    }
```